asserts/gpgkeypairmgr.go

```diff
@@ -257,7 +257,16 @@ func (gkm *GPGKeypairManager) Get(keyID string) (PrivateKey, error) {
 }
 
 func (gkm *GPGKeypairManager) sign(fingerprint string, content []byte) (*packet.Signature, error) {
-	out, err := gkm.gpg(content, "--personal-digest-preferences", "SHA512", "--default-key", "0x"+fingerprint, "--detach-sign")
+	args := func() []string { 
```

Comment:
Unless you are planning to reuse these parameter lists somewhere, I wouldn't write this as a function.

Reply:
Yep, makes sense.
Removed the function implementation.